docker-compose-setup/docker-compose.yml

```diff
@@ -0,0 +1,54 @@
+services:
+  coatrack-postgres:
+    image: postgres:9.4.26
+    container_name: coatrack-postgres
+    volumes:
+      - coatrack-postgres:/var/lib/postgresql/data
+    environment:
+      POSTGRES_PASSWORD: password1234
+  coatrack-config-server:
+    image: coatrack/coatrack-config-server:2.0.0-SNAPSHOT
+    container_name: coatrack-config-server
+    depends_on:
+      - coatrack-postgres
+    ports:
+      - 8998:8998
+    entrypoint: ["sh", "-c", "java -Djava.security.egd=file:/dev/./urandom \
+      -jar /opt/coatrack/lib/coatrack-config-server.jar \
+      --spring.datasource.url=jdbc:postgresql://coatrack-postgres/ygg_config_server \
+      --spring.datasource.password=password1234"]
+  coatrack-admin:
+    image: coatrack/coatrack-admin:2.0.0-SNAPSHOT
+    container_name: coatrack-admin
+    volumes:
+      - coatrack-admin:/home
```

Comment:
This is not clear to me:

- why do we need this volume? what will be stored there?
- why is it mapped to `/home`? usually `/home` only contains sub-folders

Reply:
The proxy jar is stored in this volume. I agree that the /home folder might be a confusing location. However, this volume becomes redundant here #38 and will be removed. So maybe we can directly merge #38 to the CoatRack main repo and this problem is solved.